packages/main/src/components/admin-header/utils/Common.jsx

```diff
@@ -0,0 +1,123 @@
+import { authAxios } from "../utils/Api";
+import styles from "../utils/styles/paymentMethod.module.css";
+
+// images
+import mastercard from "../assets/Mastercard.svg";
+import jcb from "../assets/jcb.svg";
+import visa from "../assets/visa.svg";
+import amex from "../assets/american-express.svg";
+import dinner from "../assets/dinners-club.svg";
+import discover from "../assets/discover.svg";
+
+// return the token from the session storage
+export const getToken = () => {
+  return sessionStorage.getItem("token") || null;
+};
+
+export const getUser = () => {
+  const result = JSON.parse(sessionStorage.getItem("user") || null);
+  return result;
+};
+
+// get Current Workspace
+export const getCurrentWorkspace = () => {
+  const currentWorkspace = localStorage.getItem("currentWorkspace") || null;
+  return currentWorkspace;
+};
+
+// get Current Workspace Data
+export const getCurrentWorkspaceData = () => {
+  if (getCurrentWorkspace()) {
+    authAxios
+      .get(`/organizations/${getCurrentWorkspace()}`)
+      .then(res => {
+        // console.log(res.data.data)
+        return res.data.data;
+      })
+      .catch(err => {
+        console.error(err);
+      });
+  }
+};
+
+export const ValidateCard = (name, style) => {
```

Comment:
we are removing pricing from the app

why are we requesting cards?

Reply:
Done